corehq/apps/domain/static/domain/js/encoder.js

```diff
@@ -0,0 +1,21 @@
+function HexParsr(salt1, salt2) {
```

Comment:
hqModules is already included because it's part of `style/base.html`. But yeah, wrapping it in a function is fine. It'll likely be turned into a module eventually, but it doesn't have to happen now.